src/Adapters/MockAdapter.ts

```diff
@@ -1043,21 +1043,54 @@ export default class MockAdapter
         }
     }
 
-    async getTimeSeriesData(_query: string) {
+    async getTimeSeriesData(query: string) {
+        let mockData: Array<ADXTimeSeries> = [];
         try {
             await this.mockNetwork();
+            try {
+                const listOfTimeSeries = query.split(';');
```

Comment:
can we add a comment ehre about what is going on and how this data is generated? there's a whole lot of splits and replacements happening and it'd be great to capture those in comments so we can make sense of them if we ever have to make updates or extend the data.

Reply:
I am just parsing the query to get quicktime, twinid and twinproperty to reflect on the generated mock data (I used to return a random mock data before independent from the query). So, there is nothing much to explain there but I can add the explanation as I said here.

Reply:
added comments